source/includes/v2/_brokerage-api-data-structures.md

```diff
@@ -0,0 +1,30 @@
+# Data structures
+
+## TagType (enum)
+
+Tag type of the account. Tag is additional information associated with particular account, 
+which can be specified in the blockchain transaction. This allows to use single blockchain 
+address to handle many accounts on it. This feature is available only on some blockchains: 
+(Ripple, Stellar, Eos...). In some blockchains different types of the tag are supported,
+and you have to specify exact type in this case.
+
++ `text` - text tag
++ `number` - numeric tag
+
+## WithdrawalDocument (object)
+
++ `properties` (*map<string,string>*) set of key-value pairs containing custom properties. 
```

Comment:
Link to list of the well-know properties?

Reply:
`+`